### PR TITLE
Docs: Consensus onboarding docs update

### PR DESCRIPTION
**Pull request by Jasagredo**
Add clarification for multi-user nix installations.
Also update the Consensus team members.

### CODE REVIEW

File: ouroboros-consensus/docs/Onboarding.md [hunk 1]
```diff
@@ -37,14 +37,17 @@ nix-env (Nix) 2.3.10
...
+If you are using a single-user installation of Nix, open your
+`~/.config/nix/nix.conf` and add/modify following entries
 
 ```
 substituters = https://cache.nixos.org/ https://hydra.iohk.io/
 trusted-substituters = https://cache.nixos.org/ https://hydra.iohk.io/
 trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
 ```
 
+If instead you are using a multi-user installation, add/modify said entries in `/etc/nix/nix.conf`.
```
Comment (dnadales) [inline ouroboros-consensus/docs/Onboarding.md:49]: Would it make sense to refer to the cache misses that one might run into if this is not correctly set?

**Review by dnadales — APPROVED** [review]

File: ouroboros-consensus/docs/Onboarding.md (hunk 1 above)
Comment (Jasagredo) [reply ouroboros-consensus/docs/Onboarding.md:49]: I think so. Will add it later today :+1:

**Review by amesgen — APPROVED** [review]
As said, I think we could remove `trusted-substitutors` here, but it shouldn't do any harm in any case.

```
       • trusted-substituters

         A  list  of  URLs of substituters, separated by whitespace. These are
         not used by default, but can be enabled by users of the Nix daemon by
         specifying  --option  substituters urls on the command line. Unprivi‐
         leged users are only allowed to pass a subset of the URLs  listed  in
         substituters and trusted-substituters.
```

File: ouroboros-consensus/docs/Onboarding.md (hunk 1 above)
Comment (Jasagredo) [reply ouroboros-consensus/docs/Onboarding.md:49]: Done :ok_hand:

**Pull request merged**